Index: os-nvidia-fabricmanager/systemd/nvidia-fabricmanager.service
===================================================================
--- os-nvidia-fabricmanager.orig/systemd/nvidia-fabricmanager.service
+++ os-nvidia-fabricmanager/systemd/nvidia-fabricmanager.service
@@ -9,7 +9,7 @@ PrivateTmp=false
 Type=forking
 TimeoutStartSec=240
 
-ExecStart=/usr/bin/nv-fabricmanager -c /usr/share/nvidia/nvswitch/fabricmanager.cfg
+ExecStart=-/usr/bin/nv-fabricmanager -c /usr/share/nvidia/nvswitch/fabricmanager.cfg
 
 LimitCORE=infinity
 
